server/repositories/base_repository.py

```diff
@@ -1,16 +1,44 @@
 import os
 import sqlite3
 
+from pypika import Table
+
 
 class Repository(object):
+    # tables
+    listings = Table('listings')
```

Comment:
I moved the table definitions to base repo, so we don't have to redefine them for every query.